crypto/sr25519/privkey.go

```diff
@@ -0,0 +1,95 @@
+package sr25519
+
+import (
+	"crypto/subtle"
+	"io"
+
+	"github.com/tendermint/tendermint/crypto"
+
+	schnorrkel "github.com/ChainSafe/go-schnorrkel"
+)
+
+// PrivKeySr25519 implements crypto.PrivKey.
+type PrivKeySr25519 [32]byte
```

Comment:
do you think it makes sense to create `PrivKeySr25519Size = 32` const and use it throughout the code?